modules/builds-basic-edit-buildconfig.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+// * builds/basic-build-operations.adoc
+
+[id="builds-basic-edit-buildconfig_{context}"]
+= Editing a BuildConfig
+
+You can use the *Edit BuildConfig* option in the *Builds* view of the *Developer* perspective to edit your build configurations. You can edit a `BuildConfig` using the following options:
+
+* *Form view*: This allows you to edit and configure your `BuildConfig` using the standard form fields and checkboxes.
+* *YAML view*: This allows you to edit and configure your `BuildConfig` manually with full control over the operations.
```

Comment:
Same here.

Reply:
- Thanks for this suggestion. Learnt something new today :smile: 
- The users will not be editing it using the form but have to edit through the raw YAML.
- Yes. Some of the advanced fields are not covered by the form and require the users to edit the YAML.